exercises/resistor-color-trio/canonical-data.json

```diff
@@ -0,0 +1,46 @@
+{
+  "exercise": "resistor-color-trio",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Orange and orange and black",
+      "property": "label",
+      "input": {
+        "colors": ["orange", "orange", "black"]
+      },
+      "expected": "33 ohms"
+    },
+    {
+      "description": "Blue and grey and brown",
+      "property": "label",
+      "input": {
+        "colors": ["blue", "grey", "brown"]
+      },
+      "expected": "680 ohms"
+    },
+    {
+      "description": "Red and black and red",
+      "property": "label",
+      "input": {
+        "colors": ["red", "black", "red"]
+      },
+      "expected": "2 kiloohms"
```

Comment:
smartly dodged the question of:

> should 1200 ohms be formatted as `1200 ohms` or `1.2 kiloohms`?

nice! as long as there are no inputs that would raise this question, it can be avoided. otherwise the description may have to get a bit more specific about when to apply prefixes. keep this in mind if someone tries to suggest any other inputs.

Reply:
Good observation! I indeed avoided this situation by carefully picking the input values :)